peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -12,7 +12,20 @@
   <meta property="og:type" content="book" />
   <meta property="book:release_date"
         content="{{ document.date|date:'Y-m-d' }}"/>
-  {% if document.author %}<meta property="book:author" content="{{ document.author }}" />{% endif %}
```

Comment:
I suggest we just drop this field, we're not actually doing the correct thing any way: https://ogp.me/#type_profile